doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,209 @@
+branch, typically a `master` branch, but also other Pachyderm
+branches. By default, Pachyderm mounts the `master` branch. However,
+if you add a branch to the name of the repo, the `HEAD` of that branch
+will be mounted.
+
+**Example:**
+
+```bash
+pachctl mount images --repos images@staging+w
+```
+
+You need to add the `+w` to the branch to enable write access.
+
+Also, you can mount a specific commit. Because commits
+might be on multiple branches, modifying them might result in data deletion
+in the `HEAD` of the branches. That is why you can only mount commits in
+read-only mode.
+
+## Mounting Repositories in Read-Write Mode
+
+Running the `pachctl mount` command with the `--write` flag grants you
+write access to the mounted repositories, which means that you can
+open the files for editing and putting them back to the Pachyderm
+repository. 
+
+For example, you have an [OpenCV pipeline](../../../getting_started/beginner_tutorial/#image-processing-with-opencv)
+up and running,
+and you need to edit files in the `images` repository. Let's
+say you want to experiment with brightness and contrast
+settings in `liberty.png`. Then, you want the pipelines
+run for the updated file and see the result in the end.
+If you do not mount the `images` repo, you would have to
+first download the files to your computer, edit them,
+and then put them back to the repository. The `pachctl mount`
+command automates all these steps for you. You can mount just the
+`images` repo or all Pachyderm repositories as directories
+on you machine, edit as needed, and, when done,
+exit the `pachctl mount` command. Upon exiting the `pachctl mount`
+command, Pachyderm uploads all the changes to the corresponding
+repository.
+
+If someone else modifies the files while you are working on them
+locally, their changes will likely be overwritten when you exit
+`pachctl mount`. This happens because your changes are saved to
+the Pachyderm repository only after you interrupt the `pachctl mount`
+command. Therefore, make sure that you do not work on the
+same files while someone else is working on them.
+
+!!! warning
+    Use writable mount **ONLY** when you have sole ownership
+    over the mounted data. Otherwise, merge conflicts or
+    unexpected data overwrites can occur.
+
+Because output repositories are created by the Pachyderm
+pipelines, they are immutable. Only a pipeline
+can change and update files in these repositories. If you try to change
+a file in an output repo, you will get an error message.
+
+## Prerequisites
+
+You must have the following configured for this functionality to work:
+
+* Unix or Unix-like operating system, such as Ubuntu 16.04 or macOS
+Yosemite or later.
+* FUSE for your operating system installed:
+
+  * On macOS, run:
+
+    ```bash
+    brew cask install osxfuse
+    ```
+
+    ```bash
+    brew install sshfs
+    ```
+
+  * On Ubuntu, run:
+
+    ```bash
+    sudo apt-get install sshfs
+    ```
```

Comment:
still need to remove this

Reply:
I'm not sure if the diff changed or I misread, but the comment I left was for a `brew install sshfs` instruction. Generally I think it makes sense to keep the macOS instructions even if our fuse support is pretty poor. We'll hopefully fix that in the future.

Reply:
@ysimonson I thought I did remove just `brew install sshfs` only. The macOS instructions are above the Ubuntu instructions. Is something missing there?

Reply:
Nope, I'm clarifying that the first comment in this thread is no longer relevant